xcdat/regridder/xesmf.py

```diff
@@ -40,13 +40,27 @@ def __init__(
         extrap_dist_exponent: Optional[float] = None,
         extrap_num_src_pnts: Optional[int] = None,
         ignore_degenerate: bool = True,
-        **options,
+        **options: Any,
     ):
         """Extension of ``xESMF`` regridder.
 
-        The ``XESMFRegridder`` extends ``xESMF`` by automatically constructing
-        the ``xe.XESMFRegridder`` object, preserving source bounds and
-        generating missing bounds.
+        This method extends ``xESMF`` by automatically constructing by
+        ``xe.XESMFRegridder`` object and ensuring bounds and metadata are
```

Comment:
Should this be `xesmf.XESMFRegridder`? (If so, it would apply elsewhere, too).

Reply:
This should actually be `xesmf.Regridder`, good catch.